README.md

```diff
@@ -274,6 +274,38 @@ package "runtime-lib" ( ... )
 You can check that this META works for all the 4 methods described
 above.
 
+### Building rewriter that you are currently developing
+When developing a new rewriter you are very likely to prepare a few
+tests for it. The compilation line above doesn't suit this task very well
+(because ocamlfind package with your rewriter is not yet installed) 
+and it will be more convenient to specify `.cm[x]a` with your rewriter
+manually. With this approach we sacrifice compilation speed using 
+`-ppx argument` option but it should not be critical because it's very
+likely that few rewriters will be used in the code of tests.
+
+For example, the standalone rewriter (`pp_foo.native`) will
+have this code
+
+    let () = Ppxlib.Driver.standalone ()
+
+in `pp_foo.ml` and will require a few extra switches to compile 
+(N.B. order matters)
+
+    ocamlfind ... ppx/ppx_foo.cmxa -package ppxlib
```

Comment:
It might be worth mentioning that `ppx_foo.cmxa` must be linked with `-linkall`, or alternatively use `ppx_foo.cmx`.